admin-manual/security/csp-headers.rst

```diff
@@ -0,0 +1,263 @@
+===============================
```

Comment:
Would be good to add an anchor to the main title page - we tend to link throughout the docs to anchors rather than pages, so if we move pages around something doesn't break. I would suggest: 

```
.. _security-csp-headers:
```

You will also need this ref to be able to link on the docs homepage, which it looks like you forgot to add in this PR!  (i.e. `atom-docs/index.rst`, under the security section of the Admin part)

Reply:
Added.